### DISCUSSION TITLE
Don't pause current playing song when opening other albums in folder structure view

### DISCUSSION THREAD

**Discussion by Groxxxy** [TT-ReBORN/Georgia-ReBORN#41] [Q&A] [answered]
I may be playing any song but choose to browse other albums in my library but when I go to look for the next song in a different album within folder structure view the current song pauses. It unpauses when I collapse the album I am looking at and pauses when I open that new album group.

Can it be that there is no pause please? I may be undecided which album I listen to next when I am only perusing my collection and wish to continue to listen to the current playing song.

Thanks! Great theme, good work!

![Z52ZvHCxEI](https://user-images.githubusercontent.com/14025275/146040881-c5aed8d5-41ec-4595-9d76-a0d7c1b7400e.gif)

**Comment by TT-ReBORN** [comment]
This seems like a bug, what monitor resolution are you using?

**Comment by Groxxxy** [comment]
2560 x 1440, just a 27" monitor.

While I followed up to your resolution inquiry I found the problem!

This bug only happens with 4K mode enabled. Disabling it prevents it!

Please see:
![g4Ev9JqakF](https://user-images.githubusercontent.com/14025275/146107802-8291d3b5-8f18-4f35-8096-d8cca5d2ce6e.gif)

**Comment by Groxxxy** [comment]
Well done TT, that has solved the issue!

![jUOxkOnJSd](https://user-images.githubusercontent.com/14025275/146200821-6a49ee05-a255-4129-bb98-60f3471eb9b6.gif)

**Comment by TT-ReBORN** [comment, marked as answer]
Thanks for reporting and testing! Fixed here 6def222.
